routing/pathfind_test.go

```diff
@@ -2,6 +2,7 @@ package routing
 
 import (
 	"bytes"
+	"crypto/sha256"
```

Comment:
Commit message nit on this: the title itself could use a bit more detail. Simply from "unit test added" one can't gain any knowledge w.r.t exactly what's being tested in the additional set of unit tests.

Reply:
Squashed unit test commit. I prefer implementation and covering test to be part of the same commit. But otherwise valid point about commit message.